movici_simulation_core/json_schemas/models/rules.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "/rules/1.0.0",
+  "type": "object",
```

Comment:
We could use this jsonschema to do (parts of) the config validation. And then we can use `movici_simulation_core.validate.ensure_valid_config` to validate the model config